api/app/api.py

```diff
@@ -1,22 +1,28 @@
 from fastapi import FastAPI, Request
+from decouple import config
 from fastapi.middleware.cors import CORSMiddleware
 from fastapi.encoders import jsonable_encoder
 from fastapi_mail import FastMail, MessageSchema,ConnectionConfig
 import json
 from app import EMAIL, PSWRD
 import os
 
+
+
 app = FastAPI()
 
+EMAIL = os.getenv("EMAIL")

```

Comment:
If you are using __init__ file to get the env variables you can import them directly, why re-fetch them?